ee/ucp/kubernetes/storage/use-azure-disk.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Configuring Azure Disk Storage for Kubernetes
+description: Learn how to add persistent storage to your Docker Enterprise clusters running on Azure with Azure Disk. 
+keywords: Universal Control Plane, UCP, Docker EE, Kubernetes, storage, volume
+redirect_from:
+---
+
+Operators can provide persistent storage for workloads running on Docker
+Enterprise when running on Microsoft Azure by using Azure Disk. Operators can
+either pre-provision Azure Disks to be consumed by Kubernetes Pods, or can use
+the Azure Kubernetes integration to dynamically provision Azure Disks on demand.
+
+
+## Pre-Requisites
+
+This guide assumes you have already provisioned a UCP environment on to
+Microsoft Azure. The Cluster must be provisioned after meeting all of the
+pre-requisites within [Install UCP on
+Azure](/ee/ucp/admin/install/install-on-azure.md).
+
+Additionally the following guide is using the Kubernetes Command Line tool `$
+kubectl` to provision Kubernetes objects within a UCP cluster. Therefore this
+tool needs to be downloaded, along with a UCP client bundle. For more
+information on configuring CLI access to UCP see [CLI Based
+Access](/ee/ucp/user-access/cli/).
+
+## Manually Provisioning Azure Disks
+
+An operator can use existing Azure Disks or manually provision new ones to
+provide persistent storage for Kubernetes Pods. Azure Disks can be manually
+provisioned in the Azure Portal, using ARM Templates or using the Azure Cli. In
```

Comment:
Should adjust instances of `Cli` to `CLI` per [Azure's style](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) and to match the rest of `CLI` in the doc